### DISCUSSION TITLE
Documentation Improvement - Capturing Notes

### DISCUSSION THREAD

**Discussion by jayrajput** [ahyatt/ekg#39] [General]
[Capturing Notes](https://github.com/ahyatt/ekg/blob/main/doc/ekg.org#7-capturing-notes) section suggest multiple methods to capture notes. But my research shows that actually there is only one way to capture notes which is `ekg-capture`. The other two methods are mere wrappers over the `ekg-capture`. The other two method shall be listed as convenience functions and probably split into it's own section. Some more details on `ekg-capture` and then relating how convenience functions are using `ekg-capture` will be great.

I feel that convenience functions opens up a whole new level of ekg usage and so want to highlight that in the documentation.

**Comment by ahyatt** [comment]
I just pushed a change to the `develop` branch that mentions this.  I didn't want to do this in the main part of the manual, since that is user-focused, so I mentioned the usefulness of wrapping `ekg-capture` in the "Integrating with ekg" section.  Thank you for the suggestion!

**Reply by jayrajput** [reply]
Thanks.
